.venv/lib/python3.11/site-packages/PIL/ImageFont.py

```
@@ -0,0 +1,1338 @@
+        self.path = font
+        self.size = size
+        self.index = index
+        self.encoding = encoding
+
+        try:
+            from packaging.version import parse as parse_version
+        except ImportError:
+            pass
+        else:
+            if freetype_version := features.version_module("freetype2"):
+                if parse_version(freetype_version) < parse_version("2.9.1"):
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)